### PR TITLE
Updates styling

### PR DESCRIPTION
**Pull request by jonnilundy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,57 +3,76 @@
...
-			font-size: 72px;
-		}
-		h2 {
-			font-size: 24px;
-		}
-	}
-}
-	</style>
+  <link rel="stylesheet" type="text/css" href="https://cdn.wedeploy.io/fonts/Circular-Std.css">
+	<title>My site is live!</title>
```
Comment (zenorocha) [inline index.html]: Indentation

File: index.html [hunk 2]
```diff
@@ -3,57 +3,76 @@
...
+  <style type="text/css">
```
Comment (zenorocha) [inline index.html]: You don't need  `type="text/css"` anymore

File: index.html [hunk 3]
```diff
@@ -3,57 +3,76 @@
...
-		h1 {
-			font-size: 72px;
-		}
-		h2 {
-			font-size: 24px;
-		}
-	}
-}
-	</style>
+  <link rel="stylesheet" type="text/css" href="https://cdn.wedeploy.io/fonts/Circular-Std.css">
```
Comment (zenorocha) [inline index.html]: You don't need  `type="text/css"` anymore

File: index.html [hunk 4]
```diff
@@ -3,57 +3,76 @@
...
+      -webkit-font-smoothing: antialiased;
+      color: rgb(14,20,26);
+      font-size: 46px;
+      font-weight: 900;
+      letter-spacing: 0;
+      line-height: 60px;
+      margin: 6px 0 0;
+    }
+    h2 {
+      -webkit-font-smoothing: antialiased;
```
Comment (zenorocha) [inline index.html]: You can put this on the `body` so it's not duplicated

**Comment by jonnilundy** [comment]
Thanks for that feedback @zenorocha! Updated the changes.

File: _error/400.html [hunk 5]
```diff
@@ -99,10 +99,14 @@ <h2>We couldn't find the page you were looking for.</h2>
...
-	<div class="logo">
-		<h5>WeDeploy</h5>
-		<p>Copyright © 2018 Liferay, Inc.</p>
-	</div>
+  <div class="logo">
+    <h5>WeDeploy</h5>
+    <p>Copyright © <span id="year"></span> Liferay, Inc.</p>
+  </div>
+  
+  <script type="text/javascript">
```
Comment (zenorocha) [inline _error/400.html]: No need for `type="text/javascript"`

Comment (jonnilundy) [reply _error/400.html]: Thanks!

**Comment by zenorocha** [comment]
@jonnilundy can you fix these mixed tabs and spaces?

screen shot 2018-03-05 at 2 59 01 pm

**Comment by jonnilundy** [comment]
@zenorocha nice, just changed all tabs to spaces.

**Pull request merged**